### DISCUSSION TITLE
Cannot change grad_req in imported model

### DISCUSSION THREAD

**Discussion by dmas-at-wiris** [apache/incubator-mxnet#13349] [Q&A]
I’m trying to load a pretrained model and fine tune it. Given that I’m using batch accumulation I need to change grad_req to ‘add’. However, when i do this it crashes with the following error:
> src/imperative/./imperative_utils.h:733: Check failed: vctx[i].dev_type != -1 (-1 vs. -1) Cannot decide context for node NoGradient

This issue is also found [here](https://discuss.gluon.ai/t/topic/5766), however no solution was provided.

**Comment by vdantu** [comment]
@mxnet-label-bot add [Question]

**Comment by Ishitori** [comment]
Hi @dmas-at-wiris . Is it still an issue? Could you provide your minimal reproducible example?
